baserun/instrumentation/anthropic.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+import logging
+from types import GeneratorType
+from typing import Collection, Any
+
+from anthropic.resources import Completions, AsyncCompletions
```

Comment:
What is the implication of importing this here?

Reply:
What if a user does not have anthropic installed?

Reply:
Damn nice catch